runbooks/english/01-setup.md

```diff
@@ -19,6 +19,18 @@ In the codespace's creation's page, keep in mind to **change the repository name
 
 ![](../codespace.png)
 
+Codespace environnement can take few minutes.
+
+> [!TIP]
+> If codespace load page always show, go to bottom of [Codespace page](https://github.com/codespaces/) 
+> ![](../codespace-2.png)
+>
+> If status is `Active`, that means the codespace is ready, but web page is blacked maybe due to too many call from same IP addresses (is case of codelab with too many people). You can connect your local VSCode of you computer directly to codespace environnement.
```

Comment:
```suggestion
> If status is `Active`, that means the codespace is ready, but web page is blocked maybe due to too many call from same IP addresses (in case of codelab with too many people). You can connect your local VSCode (from your computer) directly to codespace environnement.
```